pgoapi/inventory.py

```diff
@@ -45,6 +48,15 @@ def setup_inventory(self):
                 self.lucky_eggs = item_count
             pokemon_family = inventory_item['inventory_item_data'].get('pokemon_family', {})
             self.pokemon_candy[pokemon_family.get('family_id', -1)] = pokemon_family.get('candy', -1)
+            pokemon_data = inventory_item['inventory_item_data'].get('pokemon_data', {})
+            if pokemon_data.get('is_egg', False) and not pokemon_data.get('egg_incubator_id', False):
```

Comment:
its the internal ID of the incubator, checking it here on the egg tells us if the egg is currently in an incubator and thus not available